src/s2wasm.h

```diff
@@ -1193,6 +1198,42 @@ class S2WasmBuilder {
         }
       }
     }
+    if (!!startFunction) {
```

Comment:
I'm ok to change the API to make it more idiomatic if you want, let's discuss.

But in general I think Name is the right thing for the AST - we know it helps perf hugely there. And anything interfacing with that should be a Name or else we'll have constant weird mismatches of std::string here but Name there. That's what I want to avoid.